example/basic/single-layer-mapbox.html

```diff
@@ -4,7 +4,6 @@
         <title>Single layer | CARTO</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta charset="UTF-8">
-        <script src="../hide-attribution.js"></script>
```

Comment:
Why are you removing this?

Reply:
This is not used anymore since the examples are separated from the acceptance tests. This will be addressed for all the examples in the "Examples ticket".

Reply:
:+1: